src/boot/bootctl-install.c

```diff
@@ -671,18 +672,34 @@ static int install_variables(
                 return log_error_errno(r, "Cannot access \"%s/%s\": %m", esp_path, path);
 
         r = find_slot(uuid, path, &slot);
-        if (r < 0)
+        if (r < 0) {
+                if (graceful) {
+                        log_full_errno(arg_quiet ? LOG_DEBUG : LOG_INFO, r,
+                                       r == -ENOENT ?
+                                       "Failed to access EFI variables, skipping. Is the \"efivarfs\" filesystem mounted?" :
+                                       "Failed to determine current boot order, skipping: %m");
+                        return 0;
+                }
+
                 return log_error_errno(r,
                                        r == -ENOENT ?
                                        "Failed to access EFI variables. Is the \"efivarfs\" filesystem mounted?" :
                                        "Failed to determine current boot order: %m");
```

Comment:
OK. This should work, but now both the graceful and non-graceful path are almost the same. Please reformat:
```C
int level = graceful ? arg_quiet ? LOG_DEBUG : LOG_INFO : LOG_ERR;
const char *skip = graceful ? ", skipping" : "";
log_full_errno(level, r,
                          r == -ENOENT ? … : …,
                          skip);
return graceful ? 0 : r;
```

And similarly for the case below.